jetty-server/src/main/java/org/eclipse/jetty/server/CustomRequestLog.java

```diff
@@ -517,18 +523,17 @@ else if (m.group("LITERAL") != null)
     {
         public final String code;
         public final String arg;
-        public final List<String> modifiers;
+        public final List<Integer> modifiers;
         public final boolean negated;
 
         public final String literal;
 
-        public Token(String code, String arg, List<String> modifiers, boolean negated)
+        public Token(String code, String arg, List<Integer> modifiers, boolean negated)
```

Comment:
this is breaking backward compat. 
Not sure someone use this but as it's public and it's 9.4.x maybe better to keep the previous constructor and mark it deprecated.
Then remove it in 10.

Reply:
The constructors are public but `Token` is a private class, it is really intended to just be part of the internal implementation.

Reply:
haha the paradox of this public constructor and private class.
Sorry for the comment I should have checked the full class

Reply:
Ok cool, do you think I should make the methods and fields on `Token` private or package protected, if that's better style.

Reply:
no I don't think so. 
It's just the diff here didn't show the full class and I have been lazy to look by myself :) 
It's just a curiosity of the language 😄